test/run_test.py

```diff
@@ -101,6 +103,32 @@
     'distributed/rpc/test_dist_autograd_spawn',
 ]
 
+# These tests are slow enough that it's worth calculating whether the patch
+# touched any related files first.
```

Comment:
Is there a reason we don't just universally apply the determinator to everything?

Reply:
Answer below appears to be, it is faster to run them than to determine them? (That sounds like an unusually fast test file...)

Reply:
A lot of the test files run in less than 2 seconds!

Reply:
If only we could collect stats on this.....